lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -84,17 +84,8 @@ export class AuthorizationCodeClient extends BaseClient {
 
         // Validate response. This function throws a server error if an error is returned by the server.
         responseHandler.validateTokenResponse(response.body);
-        return await responseHandler.handleServerTokenResponse(
-            response.body, 
-            this.authority, 
-            reqTimestamp, 
-            request, 
-            authCodePayload,
-            undefined,
-            undefined,
-            undefined,
-            requestId
-        );
+        const httpVer = response.headers?.[HeaderNames.X_MS_HTTP_VERSION] || undefined;
+        return await responseHandler.handleServerTokenResponse(response.body, this.authority, reqTimestamp, request, authCodePayload, undefined, undefined, undefined, httpVer, requestId);
```

Comment:
With long lists of parameters, the format of one line per parameter is generally more readable.  It also gives some flexibility if you want to comment on parameters.
```
undefined,   // thing with stuff
undefined,   // other thing
undefined,   // yet another thing...
```